ntoskrnl/io/iomgr/irq.c

```diff
@@ -185,7 +187,7 @@ IopConnectInterruptExFullySpecific(
                                 Parameters->FullySpecified.ShareVector,
                                 Parameters->FullySpecified.ProcessorEnableMask,
                                 Parameters->FullySpecified.FloatingSave);
-    DPRINT("IopConnectInterruptEx_FullySpecific: has failed with status %X", Status);
+    DPRINT1("IopConnectInterruptExFullySpecific: has failed with status 0x%lx\n", Status);
```

Comment:
Despite the message, this is also printed on success. So probably shouldn't be turned into a DPRINT1

Reply:
Right. I added an `if ()`, though I'm not sure what dabe7fb author(s) intent was.